content/docs/hello-world.md

```diff
@@ -12,7 +12,7 @@ redirect_from:
   - "docs/getting-started-zh-CN.html"
 ---
 
-The easiest way to get started with React is to use [this Hello World example code on CodePen](codepen://hello-world). You don't need to install anything; you can open it in another tab and follow along as we go through examples. If you'd rather use a local development environment, check out the [Installation](/docs/try-react.html) section.
+Getting Started With React
```

Comment:
Do we ever need this heading?

Reply:
Looks a bit weird if the next like becomes the subheading

Reply:
I don't think I agree.

It's not a subheading, it's a leading paragraph. We use it this way on all the other pages. I don't think we use this style for a subheading.

Reply:
Ok